server/cmd/mmctl/commands/team.go

```diff
@@ -346,16 +348,24 @@ func modifyTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	teams := getTeamsFromTeamArgs(c, args)
 	for i, team := range teams {
 		if team == nil {
-			printer.PrintError("Unable to find team '" + args[i] + "'")
+			errText := "Unable to find team '" + args[i] + "'"
+			printer.PrintError(errText)
+			errs = multierror.Append(errs, errors.New(errText))
 			continue
 		}
 		if updatedTeam, _, err := c.UpdateTeamPrivacy(context.TODO(), team.Id, privacy); err != nil {
-			printer.PrintError("Unable to modify team '" + team.Name + "' error: " + err.Error())
+			errText := "Unable to modify team '" + team.Name + "' error: " + err.Error()
+			printer.PrintError(errText)
+			errs = multierror.Append(errs, errors.New(errText))
 		} else {
 			printer.PrintT("Modified team '{{.Name}}'", updatedTeam)
 		}
 	}
 
+	if errs != nil {
+		return errs
+	}
+
 	return nil
```

Comment:
And then this can be 
```suggestion
	return err.ErrorOrNil()
```

Reply:
Updated